dts/bindings/can/atmel,sam0-can.yaml

```diff
@@ -0,0 +1,29 @@
+description: Specialization of Bosch m_can CAN-FD controller for Atmel SAM0
+
+compatible: "atmel,sam0-can"
+
+include:
+  - name: can-fd-controller.yaml
+  - name: pinctrl-device.yaml
```

Comment:
Aren't you missing an include on `bosch,m_can-base.yaml`?

Reply:
In my opinioin - no - because `bosch,m_can-base.yaml` is defined on controller level, not in individual module.
The same way is defined for sam-can.